lib/browser/api/auto-updater/auto-updater-win.ts

```diff
@@ -15,7 +15,7 @@ class AutoUpdater extends EventEmitter {
   }
 
   getFeedURL () {
-    return this.updateURL;
+    return this.updateURL ?? '';
```

Comment:
the Mac implementation returns an empty string here if the URL is not specified. We should either do that here as well, or change the documentation to allow `null`